test/unit/products.ts

```diff
@@ -56,13 +57,25 @@ suite('Products', ({ expect, spy, stub, itShouldBeConfigurable, itShouldHaveAlia
 
     it('should mixin props to state', () => {
       const state = <any>{ a: 'b' };
+      products.updateProducts = spy();
```

Comment:
Why make this a spy if you don't check it?
`products.updateProducts = () => null;`